src/main/resources/fabric.mod.json

```diff
@@ -22,7 +22,8 @@
     ]
   },
   "depends": {
-    "fabricloader": ">=0.4.0",
-    "fabric": "*"
+    "fabricloader": ">=0.11.3",
+    "fabric": "*",
+    "java": ">=16"
```

Comment:
This should probably also require MC 1.17 snapshots and up

Reply:
You're right. Correcting it now.